### PR TITLE
wip: Start adding more golang highlights to show ts at work

### PR DESCRIPTION
**Pull request by tjdevries**
Example of some new features:

image

## Test plan

- [ ] Test to make sure new highlighting looks good
- [ ] Make sure that light theme highlighting is less aggressive (I didn't really compare much last time)

### CODE REVIEW

**Review by olafurpg — COMMENTED** [review]
Are there changes to the generated syntax kinds? I was expecting a larger diff in the snapshot tests 🤔

**Comment by tjdevries** [comment]
> Are there changes to the generated syntax kinds? I was expecting a larger diff in the snapshot tests thinking

There isn't that big of a diff in the generated syntax kinds cause I was already matching a lot of those, just being very conservative in the css.

File: docker-images/syntax-highlighter/src/bin/lsif-syntax-repl.rs [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
 use sg_syntax::DocumentFileRange;
+use sg_syntax::SourcegraphQuery;
 
 fn main() {
     println!("========================================");
     println!("  Welcome to lsif-syntax-repl");
     println!("========================================");
 
+    let mut extension = String::new();
     let contents = if let Some(path) = std::env::args().nth(1) {
```
Comment (mrnugget) [inline docker-images/syntax-highlighter/src/bin/lsif-syntax-repl.rs:19]: Nitpick (feel free to ignore), but I'd change the blocks to return tuples and use `let (contents, extension) = if let ...` and then have `String::new()` be returned in the `else` block.

**Comment by efritz** [comment]
I like this.

**Pull request closed without merging**